README.md

```diff
@@ -25,7 +25,7 @@ git clone --branch master https://github.com/ether/etherpad-lite.git && cd ether
 ```
 
 ### Manual install
-You'll need git and [node.js](https://nodejs.org) installed (minimum required Node version: **10.13.0**).
+You’ll need git and [node.js](https://nodejs.org) installed (minimum required Node version: **10.13.0**).
```

Comment:
please no.  Throughout entire project it's ``'``

Reply:
Why can’t the project be updated?

Reply:
I tend to focus on priorities that matter.  

``'`` is my style and it's the majority of the project.

I don't see value in switching away from ``'``.